src/app/content/components/popUp/FiltersList.tsx

```diff
@@ -90,10 +92,10 @@ export const FiltersListChapter = (props: FiltersListChapterProps) => (
   <FilterListItem>
     <StyledPlainButton
       aria-label={useIntl().formatMessage(
-        { id: props.ariaLabelKey },
+        { id: props.ariaLabelKey(props.title) },
         { filterValue: splitTitleParts(props.title).join(' ') }
       )}
-      data-analytics-label={`Remove breadcrumb for chapter ${splitTitleParts(props.title).join(' ')}`}
+      data-analytics-label={props.dataAnalyticsLabel(props.title)}
```

Comment:
We could probably pass here `splitTitleParts(props.title).join(' ')` to keep this logic in one place